oaknational/Oak-Web-Application
# feat: typography a tag component

## Description

- Adds typography 'A' anchor tag (styled next/link)

## How to test

1. Go to {[cloud link](https://samara-storybook-lve7br66f-oak-national-academy.vercel.app/?path=/story/typography-a--a)}
2. Click on typography section, A
3. You should see a clickable a link

## Checklist
